FindMe/Location/LocationManager.m

```diff
@@ -97,9 +109,38 @@ - (void)locationManager:(CLLocationManager *)manager didFailWithError:(NSError *
 
 - (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray<CLLocation *> *)locations
 {
-    
+    [self decodeLocation:[locations lastObject] withCompletion:^(CLPlacemark *decodedLocation) {
+        if (decodedLocation) {
+            [DatabaseManager fetchContacts:^(NSArray * _Nonnull contacts) {
+                if (contacts.count > 1) {
+                    for (Contact *contact in contacts) {
+                        [AlertManager makeStringFromPlacemarkAndSendEmail:decodedLocation withContact:contact];
```

Comment:
Probably Alert manager should not know about Locations. This is the point of location manager. Alert manager should receive the information you want to pass.